README.md

```diff
@@ -1,2 +1,4 @@
+[![CircleCI](https://circleci.com/gh/limitless-harmony/memeing-maker-backend/tree/Setup-CI-CD-CircleCi.svg?style=svg)](https://circleci.com/gh/limitless-harmony/memeing-maker-backend/tree/Setup-CI-CD-CircleCi)
```

Comment:
For the batch too, is there a way this can point to the default branch instead of hardcoding a branch here?

Reply:
I did this to see it working , will revert it to default.